### PR TITLE
Made time & address consistent

### PR DESCRIPTION
**Pull request by weirdestnerd**
# What did I do?
For the addresses that weren't the full address, I changed them to be full. I also changed the time format to "DD:DD am/pm" and made it consistent for all times.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -98,155 +98,155 @@ const App = () => {
     const PICKUPS = [
       {
           field: 'Village Park',
-          address: '6161 S Jasper Way, Centennial, C0 80016',
```
Comment (tinchonapa) [inline src/App.js:101]: The abbreviation for the state of Colorado has a zero '0' instead of 'O'

File: src/App.js [hunk 2]
```diff
@@ -98,155 +98,155 @@ const App = () => {
...
+          address: '4300 E Third Ave, Denver, CO, 80220',
+          time: '10:00 am',
           day: 'Sunday',
           contact: 'Christopher Wood',
       },
       {
           field: 'Garfield Lake Park',
-          address: '3600 W Mississippi ave, denver, CO, 80219',
-          time: '11am',
+          address: '3600 W Mississippi Ave, denver, CO, 80219',
```
Comment (tinchonapa) [inline src/App.js:157]: Capitalize Denver

File: src/App.js [hunk 3]
```diff
@@ -98,155 +98,155 @@ const App = () => {
...
+          time: '11:00 am',
           day: 'Saturday',
           contact: 'Ben Medina',
       },
       {
           field: 'Garfield Lake Park',
-          address: '3600 W Mississippi ave, denver, CO, 80219',
-          time: '4:30pm',
+          address: '3600 W Mississippi Ave, denver, CO, 80219',
```
Comment (tinchonapa) [inline src/App.js:164]: Capitalize Denver

**Comment by tinchonapa** [comment]
Hey @weirdestnerd good job by doing the boring job of going line by line and fixing the data, I just mentioned a few that you might have missed.

Also sorry, but just had the thought of what if we separate the address into fields of Address(Number & Street), City/Neighbourhood, State, Zip Code. My thought for this, and yes I should have brought it earlier, it's that it might be easier for later when we transition to a DB and need to do queries or now to filter search. We can automate the code to add the commas.

What's your thought?

**Comment by weirdestnerd** [comment]
@tinchonapa Thanks for catching what I missed. 

I think it's a great idea to atomise the address. I can add a task in Trello for it. It'll definitely help with searching.

**Comment by weirdestnerd** [comment]
@tinchonapa Can you approve again pls?

**Review by tinchonapa — APPROVED** [review]

**Pull request merged**